### PR TITLE
POM implementation for currently used pages

### PR DESCRIPTION
**Pull request by Frawless**
This PR introduces POM for most of the pages which we currently use. 

I didn't rework all of the locators as it will take much longer so this is the first step.

### CODE REVIEW

File: lib/auth.ts [hunk 1]
```diff
@@ -1,38 +1,60 @@
-import { expect, Page } from '@playwright/test';
```
Comment (kornys) [inline lib/auth.ts:1]: isn't this file duplicate of lib/pom/auth.ts?

Comment (Frawless) [reply lib/auth.ts:1]: leftover, will delete it

**Review by kornys — DISMISSED** [review]

**Review by mmusil — COMMENTED** [review]
Left a few inline comments. Suggested changes are completely optional and absolutely could be done later in some other PR.

**Review by kornys — DISMISSED** [review]

**Review by mmusil — DISMISSED** [review]
Looks good. If tests pass let's merge it.

**Review by mmusil — APPROVED** [review]
10 secs seem reasonable.

**Review by mmusil — APPROVED** [review]

**Review by kornys — APPROVED** [review]

**Review by dhlavac — APPROVED** [review]

**Pull request merged**